terraform/install.sh

```diff
@@ -283,16 +284,26 @@ loadGen() {
   gcloud container clusters get-credentials loadgenerator --zone "$LOADGEN_ZONE"
   kubectx loadgenerator=.
 
-  # find the IP of the load generator web interface
+  # Kicks of initial load generation via a POST request to Locust web interface
+  # This is because Locust currently doesn't support CLI and UI at the same time
   TRIES=0
-  while [[ $(curl -sL -w "%{http_code}"  "http://$loadgen_ip" -o /dev/null --max-time 1) -ne 200  && \
+#  while [[ $(curl -sL -w "%{http_code}"  "http://$loadgen_ip" -o /dev/null --max-time 1) -ne 200  && \
+  while [[ $(curl -XPOST -d "user_count=100&spawn_rate=10" http://$loadgen_ip/swarm -o /dev/null -w "%{http_code}" --max-time 1) -ne 200  && \
```

Comment:
is /swarm idempotent? Can we keep trying to hit it multiple times and always expect a 200 response?

Reply:
It is indempotent